src/app/containers/ArticleTimestamp/index.test.jsx

```diff
@@ -145,6 +145,30 @@ describe('ArticleTimestamp', () => {
     expect(firstChild(renderedWrapper)).toMatch(regexDate);
   });
 
+  it('should render one timestamp with date when firstPublished before today and lastPublished was published less than 1 minute after firstPublished', () => {
+    const renderedWrapper = renderedTimestamps(
+      <WrappedArticleTimestamp
+        firstPublished={1530947280000}
+        lastPublished={1530947286000}
+        delta={1}
```

Comment:
is it a requirement for this to be configurable through a prop? If not then I think it should be a constant in the component.

Reply:
It is to ensure that we don't alter the behaviour of the articles' timestamp.

Reply:
Not sure I understand

Reply:
Because the articles pages also use this component, the delta prop has been added so it doesn't change for them too. Maybe we need to talk to the stories team to see if this behavior should be the same for them as well (which I think it should).

Reply:
This fix was for a bug where `firstPublished` and `lastPublished` were slightly different values for CPS assets, even when the asset was only published once.  This caused a second "Updated on..." timestamp to appear, even when the MAP was never updated.

Optimo doesn't have this bug, and to roll the change out to articles would mean that if an Article was updated shortly after being published, we'd hide the "Updated on" notification

That would seem to be a defect - it's admittedly minor, but given it's pretty easy to avoid, it does seem worth doing.

In general, we don't want to change the product behaviour of another team without consulting them